src/Samples/Sample.AspNetCore/Controllers/CheckOutController.cs

```diff
@@ -244,15 +287,40 @@ public async Task<JsonResult> GetPaymentJsSource(PaymentInstrument instrument)
                 case PaymentInstrument.Swish:
                     var swishPayment = await CreateSwishPayment();
                     return new JsonResult(swishPayment.Operations.ViewSales.Href);
+
+                case PaymentInstrument.Trustly:
+                    var trustlyPayment = await CreateTrustlyPayment();
+                    return new JsonResult(trustlyPayment.Operations.ViewSale.Href);
                 default :
                     return null;
             }
         }
  
 
+
+        public ViewResult Aborted()
```

Comment:
Codacy found an issue: [Make 'Aborted' a static method.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)

Reply:
@asbjornu This is an MVC action, it can't be static?